gnovm/tests/imports.go

```diff
@@ -636,6 +636,29 @@ func testPackageInjector(store gno.Store, pn *gno.PackageNode) {
 				m.Context = ctx
 			},
 		)
+		pn.DefineNative("TestSkipTimestamps",
```

Comment:
skipping logic and naming came from `TestSkipHeights`

https://github.com/gnolang/gno/blob/b6dc190a60a00b63592d0198f3a86152d5d3f3b5/gnovm/tests/imports.go#L642-L656

even if it may not be good(descriptive) name, shouldn't it be better to follow `TestSkipHeights` ?

Reply:
I think TestSkipHeights makes sense, but not TestSkipTimestamps. I think if anything we can do TestSkipTime or TestSkipDuration

Reply:
TestSkipTime sounds clear